packages/docs/developer-resources/overview.md

```diff
@@ -16,8 +16,10 @@ View the [Developer Code Examples page](start.md) to get started using the Celo
   - Manage connections to the Celo blockchain, accounts, send transactions, interact with smart contracts, etc.
   - A set of wrappers around the core protocol smart contracts to easily connect with contracts related to governance, validators, on-chain exchange, etc.
   - Includes [web3.js](https://web3js.readthedocs.io/en/v1.2.4/)
-- [Celo Ethers.js Wrapper](https://github.com/celo-tools/celo-ethers-wrapper) (*experimental*)
+- [Celo Ethers.js Wrapper](https://github.com/celo-tools/celo-ethers-wrapper) (_experimental_)
   - A minimal wrapper to make [ethers.js](https://docs.ethers.io/v5/) compatible with the Celo network
+- [use-contractkit](https://use-contractkit.vercel.app)
```

Comment:
Are there docs for this? Or a link to the Github?

Reply:
Do we want to add to Community projects instead if transferrring ownership to `celotools`

Reply:
I updated the link, community_projects.md feels like dapp's only though.